packages/dso-toolkit/src/components/history-items/history-items.args.ts

```diff
@@ -0,0 +1,51 @@
+import { HandlerFunction } from "storybook/actions";
+import { ArgTypes } from "storybook/internal/types";
+import { fn } from "storybook/test";
+
+import { argTypeAction } from "../../storybook";
+
+import { HistoryItem, HistoryItems } from "./history-items.models.js";
+
+export interface HistoryItemsArgs {
+  label: string;
+  listPattern: "in-werking" | "ontwerp";
```

Comment:
verwerken in een type. Dan kan je deze ook gebruiken in `history-items.content.ts`


```suggestion
export type ListPattern = "in-werking" | "ontwerp";
```